src/storage/src/storage_state.rs

```diff
@@ -523,15 +523,25 @@ impl<'w, A: Allocate> Worker<'w, A> {
                     0,
                     "only worker #0 is doing async processing"
                 );
-                self.storage_state.internal_cmd_tx.send(
-                    InternalStorageCommand::CreateIngestionDataflow {
-                        id,
-                        ingestion_description,
-                        as_of,
-                        resume_uppers,
-                        source_resume_uppers,
-                    },
-                );
+                if resume_uppers.iter().any(|(_, f)| f.is_empty()) {
+                    tracing::info!(
+                        %id,
+                        ?as_of,
+                        ?resume_uppers,
+                        ?source_resume_uppers,
+                        "not resuming ingestion because at least one resume upper is empty"
+                    );
```

Comment:
We should resume if any resume upper is non-empty because that means there is a subsource that has progress to be made. I think you could skip sending the command if all resume uppers are empty but I worry that skipping sending the command might also skip going through the standard state transitions on all the workers (which will send DroppedId responses to the controller)